boards/arm/bt510/bt510.dts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2020 Laird Connectivity
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "Laird Sentrius BT510 Sensor";
+	compatible = "nordic,pca10056-dk";
```

Comment:
Our board is not technically compatible with the `nrf52840dk_nrf52840` (PCA10056).  Change this to `lairdconnect,bt510`

Reply:
Fixed.